.circleci/config.yml

```diff
@@ -438,32 +438,28 @@ jobs:
                 command: ./project.rb bigquerytest
 
   ui-unit-test:
-    executor: workbench-executor
+    machine:
+      image: ubuntu-2204:2022.04.1
     steps:
       - checkout-code
       - halt-test-check:
           dir_names: "ui"
-      - run:
-          name: Download Swagger CLI
-          working_directory: ~/workbench
-          command: ruby -r ./aou-utils/swagger.rb -e Workbench::Swagger.download_swagger_codegen_cli
-      - ui-cache
       - run:
           name: Yarn deps
-          working_directory: ~/workbench/ui
-          command: yarn deps
+          working_directory: ui
+          command: nvm use node; yarn deps
       - run:
           name: Run React jest tests
-          working_directory: ~/workbench/ui
+          working_directory: ui
           command: NODE_OPTIONS=--max_old_space_size=4096 yarn test --runInBand
       - run:
           name: Build UI with strict compilation
-          working_directory: ~/workbench/ui
-          command: ./project.rb build --environment test
+          working_directory: ui
+          command: CI=false ./project.rb build --environment test
```

Comment:
but we are in a CI

Reply:
Yes, this is gross, but may be necessary until the build warnings are fixed. The check for this is hardcoded into a dependency, so you can't use something explicit like, "TREAT_WARNINGS_AS_ERRORS=false."

Reply:
big improvement in this file from earlier, thanks

Reply:
Removing this will be my highest priority for marking this story as done.